di/admin_units.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Oct 22 17:53:54 2020
+
+@author: giuseppeperonato
+"""
+
+import geopandas as gpd
+import pandas as pd
+import sqlalchemy as sqla
+import utilities
+
+# GISCO datasets GEOJSON EPSG:4326 1:1milion
+datasets = {
+    "countries": "https://gisco-services.ec.europa.eu/distribution/v2/countries/geojson/CNTR_RG_01M_2020_{}.geojson",
+    "nuts": "https://gisco-services.ec.europa.eu/distribution/v2/nuts/geojson/NUTS_RG_01M_2021_{}.geojson",
+    "lau": "https://gisco-services.ec.europa.eu/distribution/v2/lau/geojson/LAU_RG_01M_2019_{}.geojson",
+}
+
+
+def get(datasets=datasets, crs="EPSG:4326"):
+    """
+    Retrieve NUTS, LAU and countries from GISCO API and make a single, consistent GDF.
+
+    Parameters
+    ----------
+    datasets : dict
```

Comment:
Those are good example of type annotation, you can specify a type directly in your method, for example:
```
def get(datasets: dict=datasets, crs: str="EPSG:4326") -> gpd.GeoDataFrame:
```

Reply:
Nice!